.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# default owners
+* @collijk @stevebachmeier @rmudambi @mattkappel @albrja @hussain-jafari @beatrixkh
```

Comment:
Interesting ordering. Maybe move to alphabetical?

Reply:
I like this idea 🙂

Reply:
Done